code/modules/mapping/helpers/mob_spawn.dm

```diff
@@ -72,108 +82,200 @@
 	var/do_damage = TRUE
 	/// If this has a value, remove a random number of organs between 0 and this max
 	var/max_organs_removed = 4
+	/// If this has a value, remove a random number of limbs between 0 and this max
+	var/max_limbs_removed = 1
 
 	/// If TRUE we delete the contents of the backpack after spawning
-	var/empty_bag = FALSE
+	var/empty_bag = TRUE
 	/// If TRUE delete the pocket contents if any
-	var/empty_pockets = FALSE
+	var/empty_pockets = TRUE
 	/// If TRUE we delete the ID slot contents after spawning
-	var/delete_id = FALSE
+	var/delete_id = TRUE
```

Comment:
you're changing default functionality here?

Reply:
Yes. Should probably specifically also mention that in the PR, but Tarmunora requested that corpse spawns should have their IDs stripped by default.
https://github.com/goonstation/goonstation/pull/18897#pullrequestreview-2056013570